modules/impersonation-system-admin-user.adoc

```diff
@@ -16,3 +16,16 @@ cluster administrator permissions.
 ----
 $ oc create clusterrolebinding <any_valid_name> --clusterrole=sudoer --user=<username>
 ----
+
+When a `system:admin` user is granted cluster administration permissions via a group, you must include the
+`*--as=<user> --as-group=<group1> --as-group=<group2>*` parameters in the command to impersonate the associated group(s).
+
+.Procedure
```

Comment:
We don't put two procedures in the same module. What you should do is add a whole new module for this new information. So basically there should be two modules here, one that tells you how to allow a user to impersonate another user, and then another module that tells you how to actually run a command, impersonating another user.

You might consider updating the title of this existing module to be more specific that it's granting permission.